testgrid/config/config.yaml

```diff
@@ -3360,9 +3362,7 @@ dashboards:
   - name: gce-kubeadm-1.6-on-1.7
     test_group_name: ci-kubernetes-e2e-kubeadm-gce-1-6-on-1-7
   - name: gce-kubeadm-upgrade-1.6-1.7
```

Comment:
Would be nice to have periodic in the name so I can tell which tests are periodic and which are ci in the grid